### ISSUE TITLE
Signup: Screen reader issues in theme picker screen

### ISSUE THREAD

**Issue by ryelle** [Automattic/wp-calypso#21656]
Testing signup with VoiceOver + Safari.

I can hear "PICK screenshot.png Bushwick, PICK screenshot.png Twenty Fifteen" etc, but these aren't read as links I can interact with. Via screen reader, my only actions here are "Back" and "Skip for now".

To fix "screenshot.png": the screenshots should either have alt text that describes the theme (not the theme name, since that's written out next to the image), or should be set to `role=presentation` to hide it completely – the former being the better solution 🙂 

To let screen reader users (and keyboard users in general) select themes, the action should be on a `Button`, rather than the link. Maybe switch "Pick" to be a compact button that appears on hover, it doesn't have to drastically change the design. The theme name should also be in the button, maybe via `ScreenReaderText` to keep the design the same.
